src/pkg/packaging/windows/targetingpack/targetingpack.wxs

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
```

Comment:
All of this appears to be boilerplate yet it is still copy/pasted in every installer?

Reply:
Yeah... I'm definitely planning to look into using some common WiX files as part of refactoring.